app/controllers/api/pods_controller.rb

```diff
@@ -7,6 +7,8 @@
 module Pod
   module TrunkApp
     class PodsController < APIController
+      MINIMUM_COCOAPODS_VERSION = Version.new(ENV['TRUNK_MINIMUM_COCOAPODS_VERSION'] || '0.36.0')
```

Comment:
Sure, you can still use 0.35, but to push you'll now need 0.36. We can always set the ENV variable on herolu.